.changeset/fluffy-moose-sin.md

```diff
@@ -0,0 +1,7 @@
+---
+"@xstate/machine-extractor": patch
+"@xstate/cli": patch
+"stately-vscode": patch
+---
+
+Fixed a bug where transition targets would not be parsed correctly if they were declared using a template literal.
```

Comment:
@Andarist this PR is based on your work. How do I mark you as the author in the changeset?

Reply:
```suggestion
author: @Andarist 

Fixed a bug where transition targets would not be parsed correctly if they were declared using a template literal.
```